action.yml

```
@@ -47,17 +47,25 @@ inputs:
     required: false
   release_label_name:
     description: The name of the label used to describe the helm release
-    default: "release"
+    default: 'release'
```

revert formatting change

```suggestion
    default: "release"
```